src/eko/quantities/couplings.py

```diff
@@ -22,7 +21,7 @@ class CouplingsInfo(DictLike):
     alphaem: Coupling
     scale: LinearScale
     max_num_flavs: FlavorsNumber
```

Comment:
It might be slightly off topic, can we drop `max_num_flavs` since it's not used anymore ?

Reply:
It's a breaking change, listed with others in #248

Reply:
yes, but we want to do it - however, maybe not in this PR, but in a new (it was also confusing in Como :upside_down_face: )

Reply:
Actually, since I already introduced a breaking change here, we can even do it here ...

Reply:
At this point, we are about to break, let's implement the whole #248.

It's not expensive implementation-wide, the problem was just breaking.

Reply:
#295